docs/src/api/optimization.md

```diff
@@ -0,0 +1,19 @@
+```@meta
+CurrentModule = RayTracer
+```
+
+# Optimization
+
+RayTracer.jl can be used for inverse rendering problems. See `examples/` for more details on
+these forms of problems.
```

Comment:
There is no link between this paragraph and optimisation, unless you know the topic well. Maybe you should add one or two sentences?